letsencrypt-auto-source/letsencrypt-auto

```diff
@@ -359,8 +359,8 @@ BootstrapMac() {
       brew install dialog
   fi
 
-  if ! hash pip 2>/dev/null; then
-      echo "pip not installed.\nInstalling python from Homebrew..."
+  if [ -z "$(brew list --versions python)" ]; then
```

Comment:
This will work since [April 2010](https://github.com/Homebrew/homebrew/commit/083e2a4b45d3f88bd5960da6caba07693e94fee4), which should be long enough for us to insist upon :)